src/Illuminate/Contracts/Foundation/ExceptionRenderer.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Illuminate\Contracts\Foundation;
+
+interface ExceptionRenderer
+{
+    /**
+     * Renders the given exception as HTML.
+     *
+     * @param  \Throwable  $throwable
+     * @return string
+     */
+    public function render($throwable);
```

Comment:
Any reason why there's no Throwable typehint here?

Reply:
I'm not sure what the rules are regarding typehints in interfaces. Some interfaces have them, other don't...